reaktive/src/commonMain/kotlin/com/badoo/reaktive/disposable/DisposableWrapper.kt

```diff
@@ -3,6 +3,7 @@ package com.badoo.reaktive.disposable
 /**
  * Thread-safe container of one [Disposable]
  */
+@Deprecated("Please use SerialDisposable", ReplaceWith("SerialDisposable"))
 @Suppress("EmptyDefaultConstructor")
 expect open class DisposableWrapper() : Disposable {
```

Comment:
`@Deprecated open class DisposableWrapper(): SerialDisposable()`

So you do not need to copy-paste implementation and still have backward compatibility.

Reply:
Yeah, I did it first, but then decided it will be a breaking change. Now I see it won't: https://wiki.eclipse.org/Evolving_Java-based_APIs_2#Evolving_API_Classes
Am I right it will not break binary compatibility?

Reply:
It won't. The signature of methods is the same. Also the class still implements `Disposable` interface. So should be fine.

Reply:
Fixed, thanks!